src/components/exchange/ExchangeMaxButton.tsx

```diff
@@ -26,28 +19,36 @@ const MaxButtonLabel = styled(Text).attrs({
   marginTop: 3,
 });
 
+interface ExchangeMaxButtonProps {
+  address: string;
+  mainnetAddress?: string;
+  disabled: boolean;
+  onPress: () => void;
+  testID: string;
+}
+
 export default function ExchangeMaxButton({
   address,
   mainnetAddress,
   disabled,
   onPress,
   testID,
-  type,
-}) {
+}: ExchangeMaxButtonProps) {
   const colorForAsset = useColorForAsset({
     address,
     mainnet_address: mainnetAddress,
-    type,
   });
   const { colors } = useTheme();
 
   return (
     <Container disabled={disabled} onPress={onPress} testID={testID}>
-      <MaxButtonContent>
-        <MaxButtonLabel color={colorForAsset || colors.appleBlue}>
-          􀜍 {lang.t('exchange.max')}
-        </MaxButtonLabel>
-      </MaxButtonContent>
+      <Inline>
+        <Box height="30px" paddingHorizontal="19px">
```

Comment:
looks like height was 32 previously, not sure what the established pattern is here -> use custom height or closest size available

Reply:
Yeah not sure what we should do for this situation, @markdalgleish whats your take?